### ISSUE TITLE
Resuming a project won't open correct last file browser view (remote folder)

### ISSUE THREAD

**Issue by acidicX** [Komodo/KomodoEdit#165]
When opening a project again, KE should resume with the last folder view that was active. Currently, KE always opens the local view, even if you were on a remote folder, which can lead to errors.

Example: I resume working on a saved project with several open files on remote. The files are opened correctly on the FTP path. So my brain assumes that I am working on the remote folder. If I then click on a file on the left side (file browser), it will open the local file and after a while I will wonder why my changes are not coming through. This happens often if I open a file after I have been working a long time on those initially opened remote files and forgot to click on the remote folder in the project after opening the project itself.

**Comment by cgchoffman** [comment]
Do you have Edit > Preferences > Workspace: "When starting Komodo" Restore last workspace enabled?

**Comment by acidicX** [comment]
Yes, I have that enabled, but the behaviour described is also the case when you switch projects - not only after opening KE.

**Comment by cgchoffman** [comment]
I assume that because the remote folder is part of the project, you've added it by using Right click project > Add > Existing remote folder?
When you open a project, Komodo will open the containing folder where the project file is saved.  We'll need to change workspaces to understand, save and open "added" folders.

**Comment by acidicX** [comment]
Yes, exactly. AFAIK there is no other way to add a remote folder to a project (I cannot save it as a remote project in the first place)?
